docs/benchmarkdotnet.md

```diff
@@ -4,29 +4,31 @@ BenchmarkDotNet is the benchmarking tool that allows to run benchmarks for .NET,
 
 ## Table of Contents
 
-- [Main Concepts](#main-concepts)
-- [Prerequisites](#prerequisites)
-- [Building the benchmarks](#building-the-benchmarks)
-  - [Using .NET Cli](#using-net-cli)
-  - [Using Python script](#using-python-script)
-- [Running the Benchmarks](#running-the-benchmarks)
-  - [Interactive Mode](#interactive-mode)
-  - [Command Line](#command-line)
-    - [Filtering the Benchmarks](#filtering-the-benchmarks)
-    - [Listing the Benchmarks](#listing-the-benchmarks)
-- [Reading the Results](#reading-the-results)
-  - [Reading the Histogram](#reading-the-histogram)
-  - [Reading Memory Statistics](#reading-memory-statistics)
-- [Multiple Runtimes](#multiple-runtimes)
-- [Regressions](#regressions)
-- [Profiling](#profiling)
-- [Disassembly](#disassembly)
-- [Private Runtime Builds](#private-runtime-builds)
-  - [Running In Process](#running-in-process)
-  - [CoreRun](#corerun)
-  - [dotnet cli](#dotnet-cli)
-  - [Private CLR Build](#private-clr-build)
-  - [Private CoreRT Build](#private-corert-build)
+- [BenchmarkDotNet](#benchmarkdotnet)
```

Comment:
this document is dedicated to BenchmarkDotNet, what do we gain by adding one more level here?

![image](https://user-images.githubusercontent.com/6011991/200819728-25ae961a-240d-498e-a09e-bfef3272414c.png)

Reply:
Fixed this, my linter automatically added it since it is a header.